.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,90 @@
+name: build-and-test
+on:
+  push:
+    branches:
+      - master
+      - iximiuz/github-actions
```

Comment:
This is probably only included for testing, right?
```suggestion
```

Reply:
Hi, Dan! 😄  Yes, this MR is still WiP and it's supposed to work in pair with [this docker-slim/examples MR](https://github.com/docker-slim/examples/pull/22).